packages/react-native/ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManager.java

```diff
@@ -1596,7 +1596,7 @@ public InspectorTargetDelegateImpl(ReactInstanceManager inspectorTarget) {
     public Map<String, String> getMetadata() {
       ReactInstanceManager reactInstanceManager = mReactInstanceManagerWeak.get();
 
-      return AndroidInfoHelpers.getInspectorHostMetadata(
+      return AndroidInfoHelpers.INSTANCE.getInspectorHostMetadata(
```

Comment:
Since these are @JvmStatic now, you shouldn't need the INSTANCE here (and anywhere else in Java files). Can you remove this?

Reply:
Sure!